app/views/projects/_list_item.html.haml

```diff
@@ -12,6 +12,11 @@
       = render :partial => "projects/like_button", :locals => {:project => project }
 :markdown
   #{truncate(project.description, length: 140)}
+
+%p
+  -if @user and membership
```

Comment:
I prefer to use `&&` instead of `and`. In this case, they have the same effect, but `and` as lower precedence and in some case you could reach into an unexpected result. I have a look at this [article](http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/).